templates/app-gateway-v2.json

```
@@ -259,11 +304,40 @@
         ]
     },
     "resources": [
+        {
+            "name": "[variables('firewallPolicyName')]",
+            "type": "Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies",
+            "apiVersion": "2020-04-01",
```

```suggestion
            "apiVersion": "2020-06-01",
```
Can this be the latest https://docs.microsoft.com/en-us/azure/templates/microsoft.network/applicationgatewaywebapplicationfirewallpolicies ?